app/viewModels/houseConsignment/index/items/PackageTypeViewModel.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels.houseConsignment.index.items
+
+import models.{CheckMode, Index, Mode, NormalMode}
+import play.api.i18n.Messages
+
+import javax.inject.Inject
+
+case class PackageTypeViewModel(mode: Mode, itemIndex: Index, houseConsignmentIndex: Index) {
```

Comment:
Think we're steadily narrowing in on a good pattern for these if you want to take a look at `NumberOfPackagesViewModel` and `PackageShippingMarksViewModel`. Makes testing a bit easier as we can mock the response in the controller spec and assert against an arbitrary heading, title etc. in the view spec. No need to worry about NormalMode/CheckMode - that logic is confined to the view model.

Reply:
That looks good, I have aligned mine.